tailwind.config.js

```diff
@@ -212,11 +212,6 @@ module.exports = {
     }
   },
   plugins: [
-    function ({ addVariant }) {
-      addVariant("child", "& > *");
-      addVariant("child-hover", "& > *:hover");
-      addVariant("child-active", "& > *:active");
-      addVariant("child-focus", "& > *:focus");
-    }
+    require("tailwindcss-radix")()
```

Comment:
Does this change cause any visual changes anywhere else in the app?

Reply:
I'm not really sure though, @0-vortex mentioned something like that -> https://github.com/open-sauced/insights/pull/705#discussion_r1058621830

Reply:
👍